lib/eventFactory.js

```diff
@@ -455,7 +455,14 @@ function updateEventParameters(config) {
         let defaultParameterValue = val;
 
         if (typeof val === 'object') {
-            defaultParameterValue = val.value;
+            if (Array.isArray(val)) {
+                defaultParameterValue = val[0];
+            } else {
```

Comment:
Do we need to add test for `else` case?

Reply:
`else` case is current behavior.
However, we missed test cases with description.
I added test cases with description.